coderd/users.go

```diff
@@ -995,9 +995,25 @@ func (api *API) organizationByUserAndName(rw http.ResponseWriter, r *http.Reques
 // @Success 201 {object} codersdk.LoginWithPasswordResponse
 // @Router /users/login [post]
 func (api *API) postLogin(rw http.ResponseWriter, r *http.Request) {
-	ctx := r.Context()
+	var (
```

Comment:
Does this handler handle all authentication paths? What if a user isn't logging with a password?

Reply:
There's a file called `userauth.go` with all of the other ways to login. This endpoint should probably be in there anyways.

Reply:
Alright, added auditing for OAuth and OIDC. I am not sure how best to smoke-test locally, but I made sure to update all the auth tests.